eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/jobs/steps/processors/SubSnpNoHgvsToVariantProcessorTest.java

```diff
@@ -190,7 +189,7 @@ public void transformSnpReverseContigRsAndSs() throws Exception {
     @Test
     public void transformInsertion() throws Exception {
         SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(1052228949L, 794529293L, "-/T", ASSEMBLY, BATCH_HANDLE, BATCH_NAME,
-                                                     CHROMOSOME, CHROMOSOME_START, CONTIG_NAME, DbsnpClass.SNV,
+                                                     CHROMOSOME, CHROMOSOME_START, CONTIG_NAME, DbsnpVariantType.SNV,
```

Comment:
You are right. The test is passing because the alleles are just treated in an special way when the variant is a STR. We don't have a variant type check and a "Variant type correct" flag now, but we may consider if this is needed.

Reply:
I have reviewed all tests variant types, and I have fixed this and other that I found it was wrong.